docusaurus.config.js

```diff
@@ -125,6 +125,14 @@ const config = {
           {
             title: 'Community',
             items: [
+              {
+                label: 'Slack - Podman Community',
+                href: 'https://app.slack.com/huddle/T08PSQ7BQ/C08MXJLCFCN',
+              },
+              {
+                label: 'Podman Development',
+                href: 'https://app.slack.com/huddle/T08PSQ7BQ/C08NTKCDC1W',
```

Comment:
Weird, when I cut/paste these URLs, I'm asked if I want to join a huddle.  I say no, then get to the page that you're pointing at.  I'm not sure where the 'huddle' bit is coming from.

Not a blocker, just noting in case others see similar or have ideas to avoid.

Reply:
DOH! I see, the huddle is buried in the URL.  I'm still unclear what the huddle is, but even less of a blocker.

Reply:
OK, I figured it out.  Please replace `huddle` with `client`
```suggestion
                href: 'https://app.slack.com/client/T08PSQ7BQ/C08NTKCDC1W',
```

But still, this won't work if they don't have access to the workspace, cloud-native.slack.com

I think with the change to client, that's OK for now and we might look at figuring out how to get "Random User" access to the workspace.